include/asterisk/res_pjsip.h

```diff
@@ -3529,7 +3531,7 @@ struct ast_sip_transport_state *ast_sip_get_transport_state(const char *transpor
 
 /*!
  * \brief Return the SIP URI of the Contact header
- * 
+ *
```

Comment:
Minor: unnecessary whitespace change here.

Reply:
Np, fixed it.